recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   - bob.extension
   - bob.blitz
   - bob.core
-  - hdf5
+  - hdf5 1.8.15*
```

Comment:
This package would be OK with   `hdf5 1.8.16`, things  would break only if installing along side `libnetcdf`.

However, when pinning `1.8.15*` we will conflict with `h5py`! So the real solution is to get our `libnetcdf` out of the door.